### ISSUE TITLE
Stuck Initializing

### ISSUE THREAD

**Issue by MarkWatt3295** [Blackymas/NSPanel_HA_Blueprint#192]
After setting up the panel it seems to get stuck on the Blue HA screen saying “initializing… Please wait”

I’ve tried a few times now but keep getting stuck with this. Pressing the small reboot button also doesn’t help.

Anything I’ve missed?

**Comment by Blackymas** [comment]
Hi - the real MarkWatt from Youtube?

Did you start with version v.3.0.0?

If the boot page stops, the boot process has not been completed. This happens when the wrong or not the most important fields are filled in the blueprint.

You can go to Trace under Automations and see where it gets stuck.

The boot process is the first row

**Comment by MarkWatt3295** [comment]
It is the real Me yes 😂.

Im creating a tutorial for this. Awesome work with the v3 update by the way.

I started a fresh panel with v3. It does just get stuck on the boot page. I will check when I’m back (I may have used a non-permitted character).

**Comment by MarkWatt3295** [comment]
Yup looks like I used a non permitted character. Working great now :)

**Comment by Blackymas** [comment]
> Yup looks like I used a non permitted character. Working great now :)

Glad to hear it works.

Sometimes life is really funny.
We were thinking of writing to you to see if you would like to test our NSPanel version and if you like our version maybe you could make a video about it. 

I flashed my first NSPanel with your NSPanel Esphome video - I really like your video ;)


I think we have built a really good and very very simple version that is easy for everyone to install and use.

If you make a video, can we link your video in the EN WIki page to help with setup and installation?

**Comment by MarkWatt3295** [comment]
> Sometimes life is really funny. We were thinking of writing to you to see if you would like to test our NSPanel version and if you like our version maybe you could make a video about it.
> 
> I flashed my first NSPanel with your NSPanel Esphome video - I really like your video ;)
> 
> I think we have built a really good and very very simple version that is easy for everyone to install and use.
> 
> If you make a video, can we link your video in the EN WIki page to help with setup and installation?

I'm working on the video now :D 
Glad to have helped with your panel and more than happy for you to feature it.
